api/controller/actionpruner/package_test.go

```diff
@@ -0,0 +1,22 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package actionpruner
+
+import (
+	"testing"
+
+	gc "gopkg.in/check.v1"
+
+	"github.com/juju/juju/api/base"
+)
+
+func TestAll(t *testing.T) {
+	gc.TestingT(t)
+}
+
+func NewPrunerFromCaller(caller base.FacadeCaller) *Facade {
+	return &Facade{
+		facade: caller,
+	}
+}
```

Comment:
nit: Ideally we shouldn't require this, but I'm not going to block the PR on it.

Reply:
yeah I just copied this from the old package